src/browser/rollbar.js

```diff
@@ -371,6 +399,14 @@ Rollbar.prototype._createItem = function(args) {
   return item;
 };
 
+Rollbar.prototype._sameAsLastError = function(item) {
+  if (this.lastError === item.err && this.lastError) {
```

Comment:
Yeah, either way for the purposes of this, but yeah the second is probably more common